flow-typed/box-ui-elements.js

```diff
@@ -688,29 +691,38 @@ type ErrorResponseData = {
 
 type ElementsXhrError = $AxiosError<any> | ErrorResponseData;
 
-type ErrorOrigins =
-    | ORIGIN_CONTENT_SIDEBAR
-    | ORIGIN_CONTENT_PREVIEW
-    | ORIGIN_PREVIEW
-    | ORIGIN_DETAILS_SIDEBAR
-    | ORIGIN_ACTIVITY_SIDEBAR
-    | ORIGIN_SKILLS_SIDEBAR
-    | ORIGIN_METADATA_SIDEBAR;
+type ElementsOrigins =
```

Comment:
This type represents a singular item, so `ElementOrigin` seems a more appropriate name.